health/notifications/alarm-notify.sh.in

```diff
@@ -1703,6 +1713,51 @@ send_awssns() {
 	return 1
 }
 
+# -----------------------------------------------------------------------------
+# Matrix sender
+
+send_matrix() {
+	local homeserver="${1}" webhook accesstoken rooms="${2}" httpcode sent=0 payload
+
+	[ "${SEND_MATRIX}" != "YES" ] && return 1
+	[ -z "${MATRIX_ACCESSTOKEN}" ] && return 1
+
+        accesstoken="${MATRIX_ACCESSTOKEN}"
```

Comment:
@okias last one - remove spaces 

And please do just a commit, it is impossible to track changes after a force-push, we have to go thgrough the whole PR again (frotunately it is not that big)

We do squash and merge, so it will be just one commit at the end of the day

Reply:
okay :)

Reply:
```bash
	[ "${SEND_MATRIX}" != "YES" ] && return 1
	[ -z "${MATRIX_ACCESSTOKEN}" ] && return 1

        accesstoken="${MATRIX_ACCESSTOKEN}"
```

^^

Reply:
yup, sorry, I missed that one and few others, fixed in next commit